### ISSUE TITLE
Outlined Select renders label with line through

### ISSUE THREAD

**Issue by juancarlosjr97** [lookfirst/mui-rff#91]
**Describe the bug**
Creating a Select component using mui-rff the label of the select component renders with the line through after select

**To Reproduce**

```
const data = [{
    label: 'Label1',
    value: 'DataLabel1'
  }]

<Select name="sector"
              label="Test"
              data={data}
              formControlProps={{
                hiddenLabel: true,
                variant: 'outlined',
                margin: 'normal'
              }}>
            </Select>
```

**Screenshots Error**

![image](https://user-images.githubusercontent.com/36451129/73030175-fe0f7680-3e30-11ea-8239-a5e416fa7315.png)

**Expected behaviour**
![image](https://user-images.githubusercontent.com/36451129/73030418-84c45380-3e31-11ea-8d74-5872bca45bfb.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version: 1.5.10

**Comment by lookfirst** [comment]
Hi @juancarlosjr97 ... thanks for the report. Do you have any ideas why that is happening? I wonder if it happens to be something to do with the 'shrink' property.

**Comment by juancarlosjr97** [comment]
Hello, thank you for your prompt reply. 

Following the example on the repository, and I can reproduce the error adding to the `formControlProps` `variant:'outlined'`

https://codesandbox.io/s/react-final-form-material-ui-example-t69gb

-

The reason is that I am not sure if `labelWidth` can be added. 

This is the working example only using material-ui

```
  const [labelWidth, setLabelWidth] = React.useState(0);

  React.useEffect(() => {
    setLabelWidth(inputLabel.current!.offsetWidth);
  }, []);

 return (
    <>
<FormControl variant="outlined" className={classes.formControl}>
              <InputLabel ref={inputLabel} id="demo-simple-select-outlined-label">
                Age
        </InputLabel>
              <Select
                labelId="demo-simple-select-outlined-label"
                id="demo-simple-select-outlined"
                // value={age}
                // onChange={handleChange}
                labelWidth={labelWidth}
              >
                <MenuItem value="">
                  <em>None</em>
                </MenuItem>
                <MenuItem value={10}>Ten</MenuItem>
                <MenuItem value={20}>Twenty</MenuItem>
                <MenuItem value={30}>Thirty</MenuItem>
              </Select>
            </FormControl>
    <>
);
```

**Comment by lookfirst** [comment]
Can you show a codesandbox with the working example that I can play with? Thanks!

**Comment by juancarlosjr97** [comment]
Sure

**Working example** 

https://codesandbox.io/s/material-demo-dbgkw

I forked an example directly from https://material-ui.com/components/selects/#select

**Comment by lookfirst** [comment]
Cool, thanks for your help. I'll take a look and play and see if I can get something working better.

**Comment by juancarlosjr97** [comment]
Thank you @lookfirst!

**Comment by lookfirst** [comment]
@juancarlosjr97 Sorry this is taking a bit of time... I've been traveling... hope to get to this soon.

**Comment by lookfirst** [comment]
@juancarlosjr97 Ok, released as 1.7.1. 

I removed the setting of the shrink, which is left over and shouldn't have been there, and I also used the same code from the MUI example that you shared.

Please let me know if anything else crops up!

**Comment by juancarlosjr97** [comment]
Hello @lookfirst 

Thank you for taking the time to look into this so quickly.

However, I am still presenting the same error and I am using the version 1.7.3

See this sandbox for more details https://codesandbox.io/s/react-final-form-select-outlined-error-t69gb

Could you let me know if there is something I am missing?

Thank you!
